style.css

```diff
@@ -0,0 +1,128 @@
+@media only screen and (max-width: 375px) {
+  .header {
```

Comment:
- [ ] OPTIONAL:  As a max-width, you are using 375px, but mobile width starts from 425px, I recommend changing max width property to 425px